app/controllers/donation_bands_controller.rb

```diff
@@ -0,0 +1,43 @@
+class DonationBandsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :find_donation_band, only: [:show, :edit, :update, :destroy]
+
+  def new
+    @donation_band = DonationBand.new
+  end
+
+  def create
+    @donation_band = DonationBand.create permitted_params
+    redirect_to :donation_bands, notice: t('donation_bands.create.notice')
+  end
+
+  def show
+    # Intentionally left blank.
+  end
+
+  def edit
+    # Intentionally left blank.
+  end
+
+  def update
+    @donation_band.update permitted_params
+    redirect_to :donation_bands, notice: t('donation_bands.update.notice')
+  end
+
+  def destroy
+    @donation_band.delete
+    redirect_to :donation_bands, notice: t('donation_bands.destroy.notice')
+  end
```

Comment:
Can we drop `destroy`? The action isn't being used, nor are we dealing with dependency issues on `page`.